### PR TITLE
Add maven badge

### PR DESCRIPTION
**Pull request by andreoss**
It makes easier to see the latest version

### CODE REVIEW

**Comment by HDouss** [comment]
@andreoss The link is pointing to org.jorigin/jeometry. This is another repository.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # jeometry
 <img align="left" alt="Logo" src="https://github.com/HDouss/jeometry/blob/gh-pages/images/path3479.png" width="100" />
 
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.github.hdouss/jeometry/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.github.hdouss/jeometry)
 [![Build Status](https://travis-ci.org/HDouss/jeometry.svg?branch=master)](https://travis-ci.org/HDouss/jeometry)
 [![Coverage Status](https://coveralls.io/repos/github/HDouss/jeometry/badge.svg?branch=master)](https://coveralls.io/github/HDouss/jeometry?branch=master)<br/><br/>
 jeometry is a Java 2D geometric figures construction library.
-It provides a simple API to:
+It provides a simple API to:com.github.hdouss
```
Comment (HDouss) [inline README.md:8]: @andreoss What is the reason behind this modification?

**Comment by HDouss** [comment]
Resolving issue #203

**Comment by HDouss** [comment]
@andreoss Can you please accept collaboration invitation so that the PR got merged by Github Workflow?

**Comment by andreoss** [comment]
@HDouss Done.

**Comment by HDouss** [comment]
@andreoss I still have his problem in Github Actions merging workflow "403 - Resource not accessible by integration". Do you have any idea how to solve it?

**Comment by HDouss** [comment]
@andreoss Seems it is a known Github Actions issue. It is not a necessarily a bug but it makes Github Actions useless for PRs that come from forked repos.

**Pull request closed without merging**